pkg/test/e2e/ccm-migration/ccm_migration_test.go

```diff
@@ -0,0 +1,153 @@
+// +build e2e
+
+/*
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ccmmigration
+
+import (
+	"context"
+	"fmt"
+	clusterv1alpha1 "github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
+	types2 "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	clusterclient "k8c.io/kubermatic/v2/pkg/cluster/client"
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
+	"k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1/helper"
+	"k8c.io/kubermatic/v2/pkg/resources"
+	e2eutils "k8c.io/kubermatic/v2/pkg/test/e2e/utils"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+	"time"
```

Comment:
please run https://github.com/xrstf/gimps, it will fix the mess here (we already have a config file for it in the repo root)